app/javascript/article-form/components/Help/EditorFormattingHelp.jsx

```diff
@@ -108,15 +108,21 @@ export const EditorFormattingHelp = ({ openModal }) => (
         </details>
       </li>
       <li>
-        You can use{' '}
+        Embed Liquid tags to add rich content such as Tweets, YouTube videos,
+        etc. Use the complete URL:&nbsp;
+        <code>{'{% embed Liquid-tag-URL %}'}</code>.{' '}
```

Comment:
I think more descriptive example would be something like:

```
{% embed https://... %}
```

Especially for those who don't really know what "liquid tags" are.

Reply:
+1